src/index.css

```diff
@@ -1 +1,25 @@
-/* #A977FE */
\ No newline at end of file
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import 'react-toastify/dist/ReactToastify.css';
+
+*,
+::before,
+::after {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+li{
+    list-style: none;
+}
+
+a{
```

Comment:
it is better to not use css on elements rather use class,id. It will get you into trouble later.